server/test/history.test.js

```diff
@@ -191,7 +191,7 @@ describe('Workflow History', function() {
   });
 
   describe('Export', function() {
-    const wfHistoryCliJson = `[{"eventId":1,"timestamp":1510701850351393089,"eventType":"WorkflowExecutionStarted","workflowExecutionStartedEventAttributes":{"workflowType":{"name":"github.com/temporalio/temporal/demo"},"taskQueue":{"name":"ci-task-queue"},"input":"eyJlbWFpbHMiOlsiamFuZUBleGFtcGxlLmNvbSIsImJvYkBleGFtcGxlLmNvbSJdLCJpbmNsdWRlRm9vdGVyIjp0cnVlfQ==","executionStartToCloseTimeoutSeconds":1080,"taskStartToCloseTimeoutSeconds":30}},{"eventId":2,"timestamp":1510701850351393089,"eventType":"WorkflowTaskScheduled","workflowTaskScheduledEventAttributes":{"taskQueue":{"name":"canary-task-queue"},"startToCloseTimeoutSeconds":180,"attempt":1}},{"eventId":3,"timestamp":1510701867531262273,"eventType":"WorkflowTaskStarted","workflowTaskStartedEventAttributes":{"scheduledEventId":2,"identity":"box1@ci-task-queue","requestId":"fafa095d-b4ca-423a-a812-223e62b5ccf8"}}]`;
+    const wfHistoryCliJson = `[{"eventId":1,"eventTime":1510701850351393089,"eventType":"WorkflowExecutionStarted","workflowExecutionStartedEventAttributes":{"workflowType":{"name":"github.com/temporalio/temporal/demo"},"taskQueue":{"name":"ci-task-queue"},"input":"eyJlbWFpbHMiOlsiamFuZUBleGFtcGxlLmNvbSIsImJvYkBleGFtcGxlLmNvbSJdLCJpbmNsdWRlRm9vdGVyIjp0cnVlfQ==","executionStartToCloseTimeoutSeconds":1080,"taskStartToCloseTimeoutSeconds":30}},{"eventId":2,"eventTime":1510701850351393089,"eventType":"WorkflowTaskScheduled","workflowTaskScheduledEventAttributes":{"taskQueue":{"name":"canary-task-queue"},"startToCloseTimeoutSeconds":180,"attempt":1}},{"eventId":3,"eventTime":1510701867531262273,"eventType":"WorkflowTaskStarted","workflowTaskStartedEventAttributes":{"scheduledEventId":2,"identity":"box1@ci-task-queue","requestId":"fafa095d-b4ca-423a-a812-223e62b5ccf8"}}]`;
```

Comment:
Apparently this JSON is not used because it should be this:

```suggestion
    const wfHistoryCliJson = `[{"eventId":1,"eventTime":"2020-07-30T00:30:03.082421843Z","eventType":"WorkflowExecutionStarted","workflowExecutionStartedEventAttributes":{"workflowType":{"name":"github.com/temporalio/temporal/demo"},"taskQueue":{"name":"ci-task-queue"},"input": {"payloads": [{"metadata": {"encoding": "anNvbi9wbGFpbg=="},"data": "IldvcmtmbG93MSI="}]},"executionStartToCloseTimeout":"1080s","taskStartToCloseTimeout":"30s"}},{"eventId":2,"eventTime":"2020-07-30T00:30:03.082421843Z","eventType":"WorkflowTaskScheduled","workflowTaskScheduledEventAttributes":{"taskQueue":{"name":"canary-task-queue"},"startToCloseTimeout":"180s","attempt":1}},{"eventId":3,"eventTime":"2020-07-30T00:30:03.082421843Z","eventType":"WorkflowTaskStarted","workflowTaskStartedEventAttributes":{"scheduledEventId":2,"identity":"box1@ci-task-queue","requestId":"fafa095d-b4ca-423a-a812-223e62b5ccf8"}}]`;
```

Reply:
it was used with thrift implementation. I kept it so we start using it too